pkgs/development/python-modules/mlx/default.nix

```diff
@@ -72,49 +66,69 @@ let
     # updates the wrong fetcher rev attribute
     passthru.skipBulkUpdate = true;
 
-    env = {
-      PYPI_RELEASE = version;
-      CMAKE_ARGS = toString [
-        # NOTE The `metal` command-line utility used to build the Metal kernels is not open-source.
-        # To build mlx with Metal support in Nix, you'd need to use one of the sandbox escape
-        # hatches which let you interact with a native install of Xcode, such as `composeXcodeWrapper`
-        # or by changing the upstream (e.g., https://github.com/zed-industries/zed/discussions/7016).
-        (lib.cmakeBool "MLX_BUILD_METAL" false)
-        (lib.cmakeOptionType "filepath" "FETCHCONTENT_SOURCE_DIR_GGUFLIB" "${gguf-tools}")
-        (lib.cmakeOptionType "filepath" "FETCHCONTENT_SOURCE_DIR_JSON" "${nlohmann_json.src}")
-        (lib.cmakeOptionType "filepath" "FETCHCONTENT_SOURCE_DIR_FMT" "${fmt_10.src}")
-      ];
-    };
+    env.PYPI_RELEASE = "1";
```

Comment:
This was always just supposed to be a boolean toggle, not something that takes a version value.